WIP 3.0 Chapters/Chapter 1 Races

```diff
@@ -4,7 +4,7 @@
   /* Resize page to international A4 */
   .phb {
     width: 210mm;
-    height: 296.8mm;
+    height: 297mm;
```

Comment:
This change should be posted up to the stylesheet globally. It's a marginal shift, but a good catch still.